package.json

```diff
@@ -29,6 +29,7 @@
     "gulp": "^3.9.1",
     "hammerjs": "^2.0.8",
     "igniteui-angular": "^5.3.0-beta.1",
+    "igniteui-angular-charts": "^5.3.0-beta.0",
```

Comment:
I suspect we may need to update this once we've published the final package?

Reply:
Yes, the update is performed every time a new package is published. We do a PR with the new package version and test that everything is OK before the update.